packages/binding-opcua/tsconfig.json

```diff
@@ -2,7 +2,10 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "outDir": "dist",
-        "rootDir": "src"
+        "rootDir": "src",
+        "skipLibCheck": true,
```

Comment:
Do we run into issues if we do the checking?

Reply:
"skipLibCheck": true required only for binding-opcua